cli/src/persist.rs

```diff
@@ -15,8 +15,8 @@ pub(crate) struct CliPersistence {
 }
 
 impl CliPersistence {
-    pub(crate) fn get_or_create_mnemonic(&self) -> Result<Mnemonic> {
-        let filename = Path::new(&self.data_dir).join(PHRASE_FILE_NAME);
+    pub(crate) fn get_or_create_mnemonic(&self, phrase_file_name: Option<&str>) -> Result<Mnemonic> {
+        let filename = Path::new(&self.data_dir).join(phrase_file_name.unwrap_or(PHRASE_FILE_NAME));
```

Comment:
If the user specifies a file shouldn't we expect a full path? It seems that the user provided file should be a path relative the the data dir.

Reply:
Applied in [8ea49ab](https://github.com/breez/breez-sdk-liquid/pull/938/commits/8ea49ab017d4c9761cbfb44b5380cce0943014d0)

Reply:
I actually meant if the user provide a file path then we should take it as is and not assume it is under the data_dir.
Does it make sense to you?